app/controllers/group_members_controller.rb

```diff
@@ -0,0 +1,72 @@
+class GroupMembersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    group = Group.find(params[:group_id])
+
+    res = current_user &&
+      (group.is_admin?(current_user) || group.is_mod?(current_user))
+    members = res ? group.members : group.members.accepted
+    members = members.page(params[:page]).per(20)
+    render json: members, meta: {cursor: 1 + (params[:page] || 1).to_i}
+  end
+
+  def create
+    membership_hash = params.require(:group_member).permit(:group_id, :user_id).to_h
```

Comment:
The reason is (iirc, it's been a few weeks) because of cases where the params are used in unsafe ways — to_h clears out all keys besides those allowed. I'll check this more in morning.